### ISSUE TITLE
mnist-inference-web example fails to run in browser with webgpu

### ISSUE THREAD

**Issue by sigma-andex** [tracel-ai/burn#1057]
**Describe the bug**

Example fails with following stack trace in browser:
```
panicked at library/std/src/sys/wasm/../unsupported/time.rs:13:9:
time not implemented on this platform

Stack:

Error
    at imports.wbg.__wbg_new_abda76e883ba8a5f (http://localhost:8000/pkg/mnist_inference_web.js:416:21)
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[488]:0x69223
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[720]:0x72eb1
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[722]:0x72f66
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[210]:0x1e5d2
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[199]:0x130be
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[334]:0x549cd
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[370]:0x5f612
    at http://localhost:8000/pkg/mnist_inference_web_bg.wasm:wasm-function[795]:0x73d96
    at __wbg_adapter_28 (http://localhost:8000/pkg/mnist_inference_web.js:208:10)
```

**To Reproduce**
```
cd /examples/mnist-inference-web
./build-for-web.sh wgpu
./run-server.sh
```

**Expected behavior**
No error.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version  119.0.6045.199

**Smartphone (please complete the following information):**

**Additional context**

**Comment by nathanielsimard** [comment]
Maybe we introduced a bug recently by enabling `autotune` by default on the `wgpu` backend, but we can't compute benchmarsk without `std` since we don't have acces to the `time` module of `std`. We could fix the feature flag in the example, but more importantly, support autotuning with `no-std`

**Comment by sigma-andex** [comment]
Fixed in #1060
